homework-11/main.js

```diff
@@ -0,0 +1,221 @@
+        },
+        {
+            productName: "Carrot",
+            productData: {
+                weight: 1000,
+                certificate: true,
+                dateOfExpiry: "30/11/2022",
+                sugarFree: true,
+            },
+            productProvider: "FOP Skyba R.R.",
+            productCountry: "UA",
+            productPrice: 30.00,
+        },
+        {
+            productName: "Chicken meat",
+            productData: {
+                weight: 1000,
+                certificate: false,
+                dateOfExpiry: "30/11/2022",
+                sugarFree: true,
+            },
+            productProvider: "FOP Skyba R.R.",
+            productCountry: "UA",
+            productPrice: 60.00,
+        },
+        {
+            productName: "Rabbit meat",
+            productData: {
+                weight: 1000,
+                certificate: false,
+                dateOfExpiry: "15/11/2022",
+                sugarFree: false,
+            },
+            productProvider: "FOP Skyba R.R",
+            productCountry: "UA",
+            productPrice: 180.00,
+        },
+        {
+            productName: "Grapefruit",
+            productData: {
+                weight: 500,
+                certificate: true,
+                dateOfExpiry: "15/11/2022",
+                sugarFree: false,
+            },
+            productProvider: "Grapefruit",
+            productCountry: "SP",
+            productPrice: 80.00,
+        },
+        {
+            productName: "Sugar",
+            productData: {
+                weight: 1000,
+                certificate: false,
+                dateOfExpiry: "15/11/2022",
+                sugarFree: false,
+            },
+            productProvider: "FOP Skyba R.R",
+            productCountry: "UA",
+            productPrice: 60.00,
+        },
+        {
+            productName: "Salt",
+            productData: {
+                weight: 500,
+                certificate: true,
+                dateOfExpiry: "30/11/2022",
+                sugarFree: false,
+            },
+            productProvider: "FOP Skyba R.R",
+            productCountry: "UA",
+            productPrice: 20.00,
+        },
+        {
+            productName: "Green Tea",
+            productData: {
+                weight: 200,
+                certificate: false,
+                dateOfExpiry: "30/11/2023",
+                sugarFree: false,
+            },
+            productProvider: "Richard",
+            productCountry: "EN",
+            productPrice: 90.00,
+        },
+    ];
+    let sum = 0;
+
+    for (let i = 0; i < listProducts.length; i++) {
+        const productItem = document.createElement('li');
+        if ((i + 1) % 2 === 0) {
+            productItem.classList.add('even')
+        }
+
+        productItem.innerHTML = `${listProducts[i].productName} <img width="10" src="img/${listProducts[i].productCountry}.png" alt="${listProducts[i].productCountry}">`;
```

Comment:
```suggestion
        productItem.innerHTML = `${listProducts[i].productName} <img src="img/${listProducts[i].productCountry}.png" alt="${listProducts[i].productCountry}">`;
```

Put styling inside CSS styles, CSS rules